Web Development/2019-20/Session 2.md

```diff
@@ -0,0 +1,26 @@
+# Session 2
+Workshop of Web Development - II 
+Conducted on: 05.09.2019
+
+## Agenda
+Organised a workshop for 1st years on JavaScript and Client-Server model
```

Comment:
A workshop for 1st years on JavaScript and Client-Server model was organised.

Use third person